k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml

```diff
@@ -61,13 +87,11 @@ spec:
               Syslog_In_Json: false
               Cluster_Name: sds-ithc00
       manager:
-        ingress:
-          annotations:
-            kubernetes.io/ingress.class: traefik
-            traefik.ingress.kubernetes.io/router.tls: "true"
```

Comment:
Why did you remove these? Is it duplicated with another level?

Reply:
Sorry which one are u referring to?

Reply:
these 4 lines